app/experimenter/experiments/serializers.py

```diff
@@ -424,6 +426,30 @@ class Meta:
         fields = ("name", "description")
 
 
+class ExperimentRecipeAddonRolloutArgumentsSerializer(serializers.ModelSerializer):
+    slug = serializers.ReadOnlyField(source="normandy_slug")
+    extensionApiId = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Experiment
+        fields = ("slug", "extensionApiId")
+
+    def get_extensionApiId(self, obj):
+        return f"TODO: {obj.addon_release_url}"
```

Comment:
I like this, should this be a branched addon thing too?

Reply:
Could do, ya.